L1Trigger/Phase2L1ParticleFlow/interface/taus/weights/b11.h

```diff
@@ -0,0 +1,25 @@
+//Numpy array shape [15]
+//Min -0.062500000000
+//Max 0.250000000000
+//Number of zeros 3
+
+#ifndef B11_H_
+#define B11_H_
+
+bias11_t b11[15] = {0.031250,
```

Comment:
Just wonder, can this kind of variable be statics? So that we can avoid the warning like
`warning: function 'TauNNIdHW::EvaluateNN' accesses or modifies non-const global static variable 'b11'.`

Reply:
@srimanob These files are provided by automatically generated firmware from the hls4ml package. In the long run the L1 team is pushing for this emulator to move way from this style of emulation which has the entire NN committed to CMSSW in this way, and towards our non accepted way of emulating this firmware which compiles it as an external to CMSSW. This file shouldn't be here in the long run.